var/spack/repos/builtin/packages/r-acepack/package.py

```diff
@@ -9,8 +9,8 @@
 class RAcepack(RPackage):
     """ACE and AVAS for Selecting Multiple Regression Transformations."""
 
-    homepage = "https://CRAN.R-project.org/package=acepack"
+    homepage = "https://cloud.R-project.org/package=acepack"
     url      = "https://cran.rstudio.com/src/contrib/acepack_1.4.1.tar.gz"
```

Comment:
Switch to https://cloud.r-project.org/src/contrib/acepack_1.4.1.tar.gz

Reply:
I did not replace any of the rstudio references but I can do another pass.

Reply:
This is fixed with the latest commit.